install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -99,66 +93,207 @@ pv0001                   <none>    5368709120   RWO           Available
+Because the group ID on the link:#nfs-export[example target NFS directory] shown
+above is 5555, the pod can define that group ID using `*supplementalGroups*`
+under the pod's global `*securityContext*` definition. For example:
+
+====
+----
+spec:
+  containers:
+    - name:
+    ...
+      #runAsUser: 99 from above has been commented out here
+  securityContext: <1>
+    supplementalGroups: [5555] <2>
+----
+<1> `*securityContext*` must be defined globally to the pod, not under a
+specific container.
+<2> An array of GIDs defined globally for the pod.
+====
+
+Assuming there are no custom SCCs that might satisfy the pod's requirements, the
+pod will likely match the *restricted* SCC. This SCC has the
+`*supplementalGroups*` strategy set to *RunAsAny*, meaning that any supplied
+group ID will be accepted without range checking.
+
+As a result, the above pod will pass admissions and will be launched. However,
+if group ID range checking is desired, a custom SCC, as described in
+link:pod_security_context#scc-supplemental-groups[pod security and custom SCCs],
+is the preferred solution. A custom SCC can be created such that minimum and
+maximum group IDs are defined, group ID range checking is enforced, and a group
+ID of 5555 is allowed.
+
+[[nfs-user-ids]]
+=== User IDs
+
+User IDs can be defined in the container image or in the pod definition.
+link:pod_security_context.html#user-id[Volume Security] covers controlling
+storage access based on user IDs, and should be read prior to setting up NFS
+persistent storage.
+
+[NOTE]
+====
+It is generally preferable to use link:#nfs-supplemental-groups[supplemental
+group IDs] to gain access to persistent storage versus using user IDs.
+====
+
+In the link:#nfs-export[example target NFS directory] shown above, the container
+needs its UID set to 99 (ignoring group IDs for the moment), so the following
+can be added to the pod definition:
+
+====
+[source,yaml]
+----
+spec:
+  containers: <1>
+  - name:
+  ...
+    securityContext:
+      runAsUser: 99 <2>
+----
+<1> Pods contain a `*securtityContext*` specific to each container (shown here)
+and a global `*securityContext*` which applies to all containers defined in the
+pod.
+<2> *nobody* owner.
+====
+
+Assuming the *default* project and the *restricted* SCC, the pod's requested
+user ID of 99 will, unfortunately, not be allowed, and therefore the pod will
+fail. The pod fails because of the following:
+
+- It requests 99 as its user ID.
+- All SCCs available to the pod are examined to see which SCC will allow a user ID
+of 99 (actually, all policies of the SCCs are checked but the focus here is on
+user ID).
+- Because all available SCCs use *MustRunAsRange* for their `*runAsUser*`
+strategy, UID range checking is required.
+- 99 is not included in the SCC or project's user ID range.
+
+It is generally considered a good practice not to modify the predefined SCCs.
+The preferred way to fix this situation is to create a custom SCC, as described
```

Comment:
The "fix this situation" implies brokenness, which isn't the intent (i believe).  Suggest: "It is generally considered a good practice to leave the predefined SCCs unmodified, and create a custom SCC..." although that doesn't sound so great, either....